src/app.module.ts

```
@@ -3,20 +3,23 @@ import { ConfigModule, ConfigService } from '@nestjs/config';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { AuthModule } from './auth/auth.module';
 import { User } from './auth/entities/user.entity';
-import { MessagingModule } from './messaging/messaging.module';
 import { Message } from './messaging/entities/message.entity';
 import { FeedModule } from './feed/feed.module';
 import { PostModule } from './post/post.module';
 import * as dotenv from 'dotenv';
 import { SavedPost } from './feed/entities/savedpost.entity';
 import { Post } from './post/entities/post.entity';
+import { MessagingModule } from './messaging/messaging.module';
 import { UserModule } from './user/user.module';
```

ESLint

[error] 'UserModule' is defined but never used.

(@typescript-eslint/no-unused-vars)